src/ExampleDynamicView.frm

```diff
@@ -70,6 +81,7 @@ Private Sub InitializeView()
         
         .LabelFor BindingPath.Create(This.ViewModel, "Instructions")
         
+        'VF: refactor free string to some enum PropertyName ("StringProperty", "CurrencyProperty") throughout (?) [when I frame a question mark in parentheses is not really a question but a rhetorical question, meaning I am pretty sure of the correct answer]

```

Comment:
That particular indirection would make it hard to later translate the enum values into actual property names, no? I don't particularly like stringly-typed things either, but without reflection or a `nameof` operator there's little we can actually do to prevent it here.